src/api-utils.ts

```diff
@@ -14,19 +14,23 @@ interface GitRef {
   };
 }
 
+interface Error {
+  status?: number;
+}
```

Comment:
I think it will be better to change it to this:
```ts
interface ErrorStatus extends Error {
    status: number;
}
```
because TypeScript already has interface Error.

Reply:
Implemented the change